notebooks/predicting_queries/simple_intent_suggestion.ipynb

```diff
@@ -0,0 +1,570 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import google.auth\n",
+    "from google.cloud import bigquery\n",
+    "import pandas_gbq\n",
+    "import nltk\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import json\n",
+    "from py2neo import Graph\n",
+    "import os\n",
+    "from ast import literal_eval\n",
+    "import json\n",
+    "from py2neo import Graph\n",
+    "from collections import Counter\n",
+    "import nltk\n",
+    "from nltk.collocations import *\n",
+    "import string\n",
+    "from scipy.stats import entropy\n",
+    "from nltk.tokenize import MWETokenizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pd.set_option('display.max_columns', 50)\n",
+    "pd.set_option('display.max_rows', 500)\n",
+    "pd.set_option('display.max_colwidth', 500)\n",
+    "PROJECT_DIR = os.getenv('PROJECT_DIR')"
```

Comment:
Need one for PROCESSED_DATA which is being called in a later part of the script:
```suggestion
    "PROJECT_DIR = os.getenv('PROJECT_DIR')"
    "PROCESSED_DATA = os.getenv('DIR_DATA_PROCESSED')"
```